org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/preferences/Preferences.java

```diff
@@ -667,8 +667,8 @@ public static Preferences createFrom(Map<String, Object> configuration) {
 			try {
 				Map<String, Object> config = (Map<String, Object>) referencedLibraries;
 				Set<String> include = new HashSet<>((List<String>) config.get("include"));
```

Comment:
this will also throw an NPE if include is null

Reply:
I thought that, since `include` is a required field, and field check is applied in vscode side, so we should just by design raise NPE if `include` field is not in the setting. 

But after some consideration I think it would be more appropriate to give default value (empty array) to include, since it would not confuse the user (by behaving like "lib/**").